tools/openvino.py

```diff
@@ -0,0 +1,156 @@
+"""Anomalib Inferencer Script.
+
+This script performs inference by reading a model config file from
+command line, and show the visualization results.
+"""
+
+# Copyright (C) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import warnings
+from argparse import ArgumentParser, Namespace
+from importlib import import_module
+from pathlib import Path
+from typing import Optional
+
+import cv2
+import numpy as np
+
+from anomalib.config import get_configurable_parameters
+from anomalib.deploy.inferencers.base import Inferencer
+
+
+def get_args() -> Namespace:
+    """Get command line arguments.
+
+    Returns:
+        Namespace: List of arguments.
+    """
+    parser = ArgumentParser()
+    # --model_config_path will be deprecated in 0.2.8 and removed in 0.2.9
+    parser.add_argument("--model_config_path", type=str, required=False, help="Path to a model config file")
+    parser.add_argument("--config", type=Path, required=True, help="Path to a model config file")
+    parser.add_argument("--weight_path", type=Path, required=True, help="Path to a model weights")
+    parser.add_argument("--image_path", type=Path, required=True, help="Path to an image to infer.")
+    parser.add_argument("--save_path", type=Path, required=False, help="Path to save the output image.")
+    parser.add_argument("--meta_data", type=Path, required=False, help="Path to JSON file containing the metadata.")
```

Comment:
We need metadata to load threshold for OpenVINO models. `required` should be set to True.